# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1259692707 28800
#      Tue Dec 01 10:38:27 2009 -0800
# Node ID 5a4f8d52bb0e0824459209713d0887d1884bda94
# Parent  83bd7f34f9dadc54f9cecc4602282ee1527ba2ba
Eliminate false dependency on conf.texi

Several files included conf.texi but did not use any of the values
defined there.  Eliminate the unneeded dependency to simplify the
build process.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-01  Rik <octave@nomad.inbox5.com>
+
+	* faq/Makefile.am, doc/interpreter/bugs.txi, doc/interpreter/install.txi:
+	Eliminate false dependency on conf.texi
+
 2009-11-30  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am:
 	Overhaul Makefile to correctly generate *.texi files and to stop their
 	distribution when running 'make dist'
 
 2009-11-30  John W. Eaton  <jwe@octave.org>
 
diff --git a/doc/faq/Makefile.am b/doc/faq/Makefile.am
--- a/doc/faq/Makefile.am
+++ b/doc/faq/Makefile.am
@@ -22,18 +22,13 @@
 TOPDIR = ../..
 
 include ../../common.mk
 
 TEXINFO_TEX = ../texinfo.tex
 
 info_TEXINFOS = OctaveFAQ.texi
 
-nodist_OctaveFAQ_TEXINFOS = ../conf.texi
-
-../conf.texi:
-	$(MAKE) -C .. conf.texi
-
 all-local: dvi html pdf ps
 
 EXTRA_DIST = OctaveFAQ.dvi OctaveFAQ.html OctaveFAQ.pdf OctaveFAQ.ps
 
 .NOTPARALLEL:
diff --git a/doc/interpreter/bugs.txi b/doc/interpreter/bugs.txi
--- a/doc/interpreter/bugs.txi
+++ b/doc/interpreter/bugs.txi
@@ -20,17 +20,16 @@
 @c distribution, as well as in the Octave manual.
 
 @ifclear BUGSONLY
 @node Trouble
 @appendix Known Causes of Trouble
 @end ifclear
 
 @ifset BUGSONLY
-@include conf.texi
 
 This file documents known bugs in Octave and describes where and how to
 report any bugs that you may find.
 
 Copyright (C) 1996, 1997, 2007 John W. Eaton.  You may copy, distribute, and
 modify it freely as long as you preserve this copyright notice and
 permission notice.
 
diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -20,17 +20,16 @@
 @c distribution, as well as in the Octave manual.
 
 @ifclear INSTALLONLY
 @node Installation
 @appendix Installing Octave
 @end ifclear
 
 @ifset INSTALLONLY
-@include conf.texi
 
 This file documents the installation of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the
 Free Software Foundation.
 
 @strong{Note:} This file is automatically generated from
